collated/test/justintkj.md

```diff
@@ -0,0 +1,568 @@
+        // different descriptor -> returns false
+        assertFalse(standardCommand.equals(new RemarkCommand(INDEX_FIRST_PERSON, VALID_REMARK_BOB)));
+    }
+}
+```
+###### \java\seedu\address\logic\commands\SortCommandTest.java
+``` java
+/**
+ * Contains integration tests (interaction with the Model) for {@code SortCommand}.
+ */
+public class SortCommandTest {
+
+    public static final String SORT_NAME_ARG = "name";
+    public static final String SORT_EMAIL_ARG = "email";
+    public static final String SORT_ADDRESS_ARG = "address";
+    public static final String SORT_NUM_ARG = "number";
+    public static final String SORT_NUM_ARG_CAMELCASE = "NuMBeR";
+    private Model model;
+    private Model expectedModel;
+    private SortCommand sortCommand;
+
+    @Before
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+    }
+    @Test
+    public void execute_showsSameList() {
+        //Sort name -> command parsed successful
+        sortCommand = new SortCommand(SORT_NAME_ARG);
+        sortCommand.setData(model, new CommandHistory(), new UndoRedoStack());
+        assertCommandSuccess(sortCommand, model, SortCommand.MESSAGE_SORT_SUCCESS + SORT_NAME_ARG, expectedModel);
+
+        //Sort email -> command parsed successful
+        expectedModel = new ModelManager(getSortedEmailAddressBook(), new UserPrefs());
+        sortCommand = new SortCommand(SORT_EMAIL_ARG);
+        sortCommand.setData(model, new CommandHistory(), new UndoRedoStack());
+        assertCommandSuccess(sortCommand, model, SortCommand.MESSAGE_SORT_SUCCESS + SORT_EMAIL_ARG, expectedModel);
+
+        //Sort address -> command parsed successful
+        expectedModel = new ModelManager(getSortedAddressAddressBook(), new UserPrefs());
+        sortCommand = new SortCommand(SORT_ADDRESS_ARG);
+        sortCommand.setData(model, new CommandHistory(), new UndoRedoStack());
+        assertCommandSuccess(sortCommand, model, SortCommand.MESSAGE_SORT_SUCCESS + SORT_ADDRESS_ARG, expectedModel);
+
+        //Sort number -> command parsed successful
+        expectedModel = new ModelManager(getSortedNumAddressBook(), new UserPrefs());
+        sortCommand = new SortCommand(SORT_NUM_ARG);
+        sortCommand.setData(model, new CommandHistory(), new UndoRedoStack());
+        assertCommandSuccess(sortCommand, model, SortCommand.MESSAGE_SORT_SUCCESS + SORT_NUM_ARG, expectedModel);
+
+        //Sort number CamelCase -> command parsed successful
+        expectedModel = new ModelManager(getSortedNumAddressBook(), new UserPrefs());
+        sortCommand = new SortCommand(SORT_NUM_ARG);
+        sortCommand.setData(model, new CommandHistory(), new UndoRedoStack());
+        assertCommandSuccess(sortCommand, model, SortCommand.MESSAGE_SORT_SUCCESS + SORT_NUM_ARG, expectedModel);
```

Comment:
The tests above are quite similar in structure. Try to minimize repetitive code blocks by extracting out similarities as utility methods.